packages/feature-toggle/src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface ExtensionConfigKeys {
+    AnnotationModeler?: string;
+    AppGenerator?: string;
+    ApplicationModeler?: string;
+    Help?: string;
+    RequirementsGathering?: string;
```

Comment:
```suggestion
    RequirementsGathering?: string;
```
Does not exist anymore